drivers/mtd/filemtd.c

```diff
@@ -708,3 +735,190 @@ bool filemtd_isfilemtd(FAR struct mtd_dev_s *dev)
 
   return (priv->mtd.erase == filemtd_erase);
 }
+
+/****************************************************************************
+ * Name: mtd_loop_register_driver
+ *
+ * Description:
+ *   Registers MTD Loop Driver
+ ****************************************************************************/
+
+#ifdef CONFIG_MTD_LOOP
+int mtd_loop_register(void)
```

Comment:
need move to public section

Reply:
It is there.... Question about private functions, they are not in private section. Should file have one private/public section or may have a few? I.e. move functions to private or start second private section?

Reply:
The whole source code just have one private and one public section, functions need move to the right place base on whether is static or global function.